codegen/templates/endpoint.tmpl

```diff
@@ -146,11 +181,19 @@ func (w {{$workflow}}) Handle(
 		return {{if eq .ResponseType ""}}nil, err{{else}}nil, nil, err{{end}}
 	}
 
+	// Filter and map response headers from client to server response.
+	endRespHead := map[string]string{}
+	{{if (ne (len $resHeaderMap) 0) -}}
+	for k, v := range {{$resHeaderMap | printf "%#v" }} {
+		endRespHead[v] = respHeaders[k]
```

Comment:
This semantics is confusing.

In the reqHeaderMap key is SERVER header and value is CLIENT header.

Here the key is CLIENT header and the value is SERVER header.

Should we do `endRespHead[k] = respHeaders[v]` ?